src/wiki/forms.py

```diff
@@ -625,10 +621,10 @@ def __init__(self, *args, **kwargs):
         self.honeypot_fieldnames = "address", "phone"
         self.honeypot_class = ''.join(
             random.choice(string.ascii_uppercase + string.digits)
-            for __ in range(10))
+            for _ in range(10))
         self.honeypot_jsfunction = 'f' + ''.join(
             random.choice(string.ascii_uppercase + string.digits)
-            for __ in range(10))
+            for _ in range(10))
```

Comment:
Because of `from django.utils.translation import gettext_lazy as _` this is double underscore